tools/uart_hw_test/README.md

```diff
@@ -22,7 +22,7 @@ Each of the above tests are performed for the following baud rates.
 115200, 38400, 19200, 9600, 4800
 
```

Comment:
You could modify the commit title as
"Add UART hardware test for Windows host with TizenRT target"

And then maybe add a couple of description lines to further describe what you add.

You can refer to previously merged commit messages.